pom.xml

```diff
@@ -146,6 +147,12 @@
 				<artifactId>joda-time</artifactId>
 				<version>${joda-time.version}</version>
 			</dependency>
+			<!-- Micrometer dependency -->
+			<dependency>
+				<groupId>io.micrometer</groupId>
+				<artifactId>micrometer-spring-legacy</artifactId>
```

Comment:
I'm a bit confused why you are pulling in the legacy support for spring boot 1.5.x. Spring Boot 2 already managed micrometer 1.0.x

Reply:
Updated pull request by removing legacy support dependency.